css/common.css

```diff
@@ -0,0 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Fjalla+One&family=Outfit:wght@200&display=swap");
+html {
+  scroll-behavior: smooth;
+  overflow-y: scroll;
+  scroll-snap-type: y;
+  scroll-padding: 15vh;
+}
+:root {
+  --red-brown: #a05941;
+  --red-light: #d0b0a6;
+  --green-black: #133032;
+  --green-light: #3e5a5b;
+  --light-grey: #ededee;
+  --white: #fff;
+  --black: #000;
+  --transparent-white: rgba(255, 255, 255, 0.75);
+  --heading-font: "Fjalla One", sans-serif;
+  --body-font: "Outfit", sans-serif;
+}
+* {
+  margin: 0;
+  box-sizing: border-box;
+  border: 0px solid black;
+  font-family: var(--body-font);
+}
+body {
+  background-color: var(--light-grey);
+  padding: 0 2rem;
+}
+.button {
+  padding: 1rem 3rem;
+  background-color: var(--red-brown);
+  text-decoration: none;
+  font-size: 1.1rem;
+  color: var(--white);
+  transition: 0.4s all ease-in-out;
+  cursor: pointer;
+}
+.button:hover {
+  color: var(--black);
+  background-color: var(--red-light);
+}
+.underline {
+  color: var(--red-brown);
+  font-size: 4rem;
+  font-weight: 600;
+  line-height: 0;
+}
+.hero {
+  width: calc(100% + 4rem);
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 -2rem;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-color: rgba(0, 0, 0, 600);
+  box-shadow: inset 0 0 0 2000px rgba(0, 0, 0, 0.25);
+}
+.hero > * {
```

Comment:
Works here, but generally best to avoid the `*` selector, other than setting very basic rules for the whole page (like `margin: 0` etc)